sklearn/linear_model/coordinate_descent.py

```diff
@@ -659,10 +659,10 @@ def fit(self, X, y, check_input=True):
 
         if (isinstance(self.precompute, six.string_types) and
            self.precompute == 'auto'):
-            warnings.warn("Setting precompute to 'auto', was found to be "
-                          "slower even when n_samples > n_features. Hence "
-                          "it will be removed in 0.18.",
-                          DeprecationWarning, stacklevel=2)
+            raise ValueError("Precompute='auto' is not supported for "
```

Comment:
precompute='auto' is still mentioned in the ElasticNet docstring above. You can probably remove it.